### ISSUE TITLE
input missing aria-label

### ISSUE THREAD

**Issue by samkelleher** [mapbox/mapbox-gl-geocoder#338]
https://github.com/mapbox/mapbox-gl-geocoder/blob/master/lib/index.js#L178

The button has an `aria-label` but the input text field itself does not.

This gets flagged in accessibility tooling.

**Comment by andrewharvey** [comment]
@samkelleher https://github.com/mapbox/mapbox-gl-geocoder/blob/master/lib/index.js#L779 does that not address this? Just trying to understand the issue here.

**Comment by samkelleher** [comment]
Thanks, it does indeed, I was running v4.5.0, and not v4.5.1 which includes the referenced line, and once updated my issue is resolved. I must have overlooked checking I was on the latest version. 👍
